modules/ardrivo/src/Wire.cxx

```diff
@@ -0,0 +1,180 @@
+#include <cstring>
+#include <fmt/format.h>
+#include <range/v3/algorithm/find_if.hpp>
+#include "Arduino.h"
+#include "BoardDataDef.hxx"
+#include "Entrypoint.hxx"
+#include "Error.hxx"
+#include "Wire.h"
+#include "WString.h"
```

Comment:
Unused include (Is also in the .h file)

Reply:
WString is just not used at all, both in the .h and the cxx. (Same with the Arduino.h include)

Reply:
Right; deleting